frontend/src/app/snapshots/page.tsx

```diff
@@ -47,13 +48,32 @@ const SnapshotsPage: React.FC = () => {
     }
 
     return (
-      <SnapshotCard
-        key={snapshot.key}
-        title={snapshot.title}
-        button={SubmitButton}
-        startAt={snapshot.startAt}
-        endAt={snapshot.endAt}
-      />
+      <div className="space-y-4">
+        <SnapshotCard
```

Comment:
Why are the releases here on this page? They were supposed to be on the page for each particular snapshot, whek you click into it from this page.
<img width="1574" height="725" alt="Screenshot 2025-08-16 at 8 16 11 PM" src="https://github.com/user-attachments/assets/9f743686-dfde-4c0c-ba2a-3d70c3fb6731" />
Please  remove them from here and update the correct spot in `app/snapshots/[id]/page.tsx`
<img width="1223" height="954" alt="Screenshot 2025-08-16 at 8 17 57 PM" src="https://github.com/user-attachments/assets/b60991e3-2835-4769-9941-61a8816a2577" />